scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -135,9 +135,9 @@ Once the VAT table is created and a tsv is exported, the AoU research workbench
     - Specify the same `call_set_identifier`, `dataset_name`, `project_id`, `extract_table_prefix`, and `interval_list` that were used in the `GvsPrepareRangesCallset` run documented above.
     - Specify the `interval_weights_bed` appropriate for the PGEN / VCF extraction run you are performing. `gs://gvs_quickstart_storage/weights/gvs_full_vet_weights_1kb_padded_orig.bed` is the interval weights BED used for Quickstart.
     - For both `GvsExtractCallset` and `GvsExtractCallsetPgenMerged`, select the workflow option "Retry with more memory" and choose a "Memory retry factor" of 1.5
+    - `GvsExtractCallset` currently defaults to 1000 alt alleles maximum, which means that any sites having more than that number of alt alleles will be dropped. Make sure to specify the appropriate `maximum_alternate_alleles` value (currently 100).
```

Comment:
I'm confused. "currently 100?"

Reply:
This is carried over from line 140. The value of max alt alleles we were told to use for Echo was 100, but in my testing we seem to be able to go to 1000 at least without `WeightedSplitIntervals` breaking down. I don't know if variants with > 100 alt alleles are actually useful though.

Reply:
for posterity's sake, I'll clarify that WeightedSplitIntervals does not actually check alleles at all.  I know we discussed this and everyone involves knows now, but juuuuust in case someone goes back to this PR in the future I want it to be stated here too lol

Reply:
I do agree with George I think that there are two values for currently in this sentence.

currently defaults to 1000
_and_ 
(currently 100).